sympy/vector/coordsys.py

```diff
@@ -23,7 +23,29 @@ def __new__(cls, name, location=None, rotation_matrix=None,
         ==========
 
         name : str
-            The name of the new CoordSysCartesian instance.
+            The name of the new CoordinateSystem instance.
+
+        system : str or Lambda
+            The transformation set from cartesian coordinate system to another.
+            It is ALWAYS assumed that the transformation set is from Cartesian.
+            Process can be slow for less common coordinate systems, use
+            built-in systems when you can.
+
+            built-in : 'cartesian', 'spherical', 'cylindrical',
```

Comment:
It seems that this and the following lines are not rendered as expected. Apparently Sphinx does not know what should be done.